src/DomHandler/Wsdl/Tag/AbstractTagOperationElement.php

```diff
@@ -36,7 +36,7 @@ public function getAttributeMessageNamespace()
         return $this->hasAttribute(self::ATTRIBUTE_MESSAGE) ? $this->getAttribute(self::ATTRIBUTE_MESSAGE)->getValueNamespace() : '';
     }
     /**
-     * @return TagMessage
+     * @return AbstractTag|null
```

Comment:
As for TagOperation, it should returna TagMessage thanks to the `'message'` parameter and the fact that [`WsdlToPhp\PackageGenerator\DomHandler\Wsdl\AbstractDocument::getElementHandler`](https://github.com/WsdlToPhp/PackageGenerator/blob/develop/src/DomHandler/Wsdl/AbstractDocument.php#L76) method uses the class corresponding to the tag name